graphql/resolvers.js

```diff
@@ -0,0 +1,130 @@
+require("dotenv").config();
+const { gql, UserInputError } = require("apollo-server");
+const jwt = require("jsonwebtoken");
+
+const operations = require("../data/operations");
+const users = require("../data/users");
+
+const resolvers = {
+  Query: {
+    getAccountAndMovements: () => operations,
+    me: (root, args, context) => {
+      return context.currentUser;
+    },
+  },
+  Mutation: {
+    addDeposit: (_, args) => {
+      const { number, type, amount } = args;
+      const findNumberAccount = operations
```

Comment:
Why do you do all this mapping? 🤔.

These resolvers should be simpler, something like:

```
accountsRepo.deposit(accountId, amount);
```

And you should hide the logic behind the data management in that accountsRepo (name is not important, just the idea).